sycl/ReleaseNotes.md

```diff
@@ -1,52 +1,687 @@
+
+- Prevented unnecessary instantiation of internal utility kernels under the hood
+  of `fill` and `memcpy` APIs if they are unused. intel/llvm#17743
+- Fixed a bug preventing the Hello, World example from the documentation being
+  built. intel/llvm#18401
+
+### Sanitizers
+
+- Extended address sanitizer and memory sanitizer to work with features that
+  require L0 cooperative dispatch (such as root-group). intel/llvm#18198
+- Fixed an undefined behavior in address and memory sanitizers implementations.
+  intel/llvm#19063
+- Fixed a bug in address and memory sanitizers which caused sporadic false
+  positive reports. intel/llvm#18253 intel/llvm#18525
+- Fixed a bug where sanitizers could segfault after reporting an error they had
+  found. intel/llvm#19009
+- Fixed a bug preventing sanitizers to be used together with shared libraries.
+  intel/llvm#18819
+- Fixed some memory leaks in sanitizers implementation. intel/llvm#19066
+- Fixed possible hangs when an application is stopped by a sanitizer because
+  errors were discovered. intel/llvm#19085
+
+- 97f508bd257d [DevSAN] Set maximum supported local/private shadow memory size (#19465)
+  -  What if the limit we set is too small? What is a user-visible side effect of this?
+
+- 69c3b9f9e01b [SYCL] Cherry-pick sanitizer patches (#19867)
+  -  Looks like 2/3 were fixes for regressions, as such the plan is to omit this
+- fae8cc2630b5 [DeviceSanitizer] Print log when build/link fail (#17521)
+- 55416cae415f [DeviceSanitizers] Set names to nameless globals (#17599)
+
+#### Address sanitizer
+
+- Improved error messages which are logged if enqueue kernel failed with
+  address sanitizer enabled. intel/llvm#20237
+- Fixed a memory leak and potential undefined behavior that may happen in
+  address sanitizer when application exits. intel/llvm#18612
+
+#### Thread sanitizer
+
+- Fixed a bug in thread sanitizer which may result in false positives reported
+  on subsequent kernel launches. intel/llvm#19355
+- Improved thread sanitizer performance by avoiding unnecessary device -> host
+  synchronization if there are no reports to transfer. intel/llvm#19133
+- Added support for detecting data races in local memory. intel/llvm#18718
+- Refined thread mapping algorithm for better performance of the thread
+  sanitizer. intel/llvm#19010
+- Added support for detecting data races in SYCL buffers. intel/llvm#17625
+- Fixed false positive reports coming from joint matrix usage. intel/llvm#18773
+- Added support for thread sanitizer in AOT mode. intel/llvm#18130
+- Fixed false positive reports coming from barriers usage. intel/llvm#17900
+- Fixed memory leaks in thread sanitizer. intel/llvm#18880
+- Improved thread sanitizer performance. intel/llvm#19127
+
+- 1d7e9ef8e71a [DevTSAN] Fix missing intercepted API and release local shadow (#18942)
+  -  Looks like some bugfix
+- e2968564552a [DevTSAN] Don't insert cleanup instruction for dynamic allocas (#18090)
+  -  Unclear what this is
```

Comment:
It's a bugfix to fix segfault caused by wrong instrumentation for dynamic allocas.